CHANGES.rst

```diff
@@ -58,6 +58,7 @@ Improvements
 - Add event logging to opening/closing registration forms, approval/rejection of
   registrations, and updates to event layout (:issue:`4360`,
   thanks :user:`giusedb` & :user:`omegak`)
+- Add editing module to event cloning (:issue:`4319`)
```

Comment:
We don't need a changelog entry here since the whole editing module is new in 2.3